### PR TITLE
Add an option to enable anti-aliasing for SVG format.

### PR DESCRIPTION
**Pull request by vincentcouvert**
@davidcl does it seem ok to you (code and documentation about settings)?

### CODE REVIEW

**Comment by mottelet** [comment]
Why not making the default antialiasing to "on" for svg output ?

File: scilab_kernel/kernel.py [hunk 1]
```diff
@@ -295,6 +296,13 @@ def _handle_svg(self, filename):
             im.data = self._fix_svg_size(im.data)
         except Exception:
             pass
+        try:
+            settings = self.plot_settings
+            print(settings)
```
Comment (davidcl) [inline scilab_kernel/kernel.py:301]: looks like a debug print, would you like to keep it ?

Comment (vincentcouvert) [reply scilab_kernel/kernel.py]: You're right, fixed f0afba96b186d39400ca55f29a74341ab1b3b213

**Comment by vincentcouvert** [comment]
> Why not making the default antialiasing to "on" for svg output ?

Because, anti-aliasing on horizontal lines make them larger than vertical lines (in axes for example):
image

I think this must be fixed before activating this option by default.

**Comment by mottelet** [comment]
I think that this is rather a browser+platform dependent issue. Have you compared the rendering between Chrome, firefox and IE ?

**Comment by vincentcouvert** [comment]
> I think that this is rather a browser+platform dependent issue. Have you compared the rendering between Chrome, firefox and IE ?

Of course! All of them display the same large horizontal lines...

**Comment by davidcl** [comment]
> Why not making the default antialiasing to "on" for svg output ?
@mottelet this is now done by default and customizable using `%plot --antialiasing False`

> Of course! All of them display the same large horizontal lines...
@vincentcouvert fixed in the latest iteration with style attribute patching for polylines

**Comment by mottelet** [comment]
Great !

**Pull request merged**